docs/src/pages/components/tooltips/SimpleTooltips.js

```diff
@@ -27,6 +27,11 @@ export default function SimpleTooltips() {
           <DeleteIcon />
         </IconButton>
       </Tooltip>
+      <Tooltip disabled title="Delete">
```

Comment:
I don't think that we should document it here. Better have a dedicated demo or nothing at all.

Reply:
Yes, I thought that too... I will remove this, I think that is not necessary.